fmriprep/workflows/base.py

```diff
@@ -311,6 +311,24 @@ def init_single_subject_wf(subject_id):
     if anat_only:
         return workflow
 
+    fmap_estimators = None
+    if "fieldmaps" not in config.workflow.ignore:
+        from sdcflows.utils.wrangler import find_estimators
+
+        # SDC Step 1: Run basic heuristics to identify available data for fieldmap estimation
+        # For now, no fmapless
```

Comment:
Are we getting rid of SDC-SyN for 20.1?

Reply:
No, the plan is to include it in after this is merged (and we cut an RC)

Reply:
Ah, okay.